### ISSUE TITLE
Villager Trading Station not working?

### ISSUE THREAD

**Issue by D4rktec** [TeamAOF/All-of-Fabric-6#205]
Hi,

i have a problem with the villager trading station. The villagers always stand around the thing but don't want anything to do with it. Is there a requirement for them to act? clear that the items to be traded must be on the input side and the "quotes" on the left side but else?

**Comment by joex92** [comment]
Following what the mod's dev explained: https://www.curseforge.com/minecraft/mc-mods/villager-trading-station
it still doesn't work, I click the trade with a paper and nothing happens... 🤔

**Comment by joex92** [comment]
Found the issue, for it to work the mod `Item Scroller` should have the Villager Trading Feature Off
![image](https://github.com/TeamAOF/All-of-Fabric-6/assets/645579/f3ac06a3-a4d4-46a9-a3d3-5e49e584ec2e)

**Comment by joex92** [comment]
Seems like `Item Scroller` uses the same method of interaction with trades as the `Villager Trading Station` mod, so for some reason `Item Scroller` is given higher priority, therefore the `Villager Trading Station` mod can't interact with trades...

**Comment by libjared** [comment]
For me the trading station only attracts villagers during the villager daily restocking time(s?).

**Comment by joex92** [comment]
yes, it works like that... and makes sense... because only in the villagers `working time` they restock their trades, the only way to make it "infinite" is that they wouldn't have a "working cycle" or if their "working cycle" last all day without resting... 🤷🏼‍♂️
